analyses/molecular-subtyping-LGAT/01-subset-files-for-LGAT.Rmd

```diff
@@ -313,13 +322,47 @@ lgat_subtypes
 
 ```
 
+Save to subset folder
+
 ```{r}
 
-# save to subset folder
 write_tsv(lgat_subtypes,file.path(subset_dir, "LGAT_snv_subset.tsv"))
 
 ```
 
+### High confidence methylation classification subtypes
+
+```{r}
+lgat_methyl_subtypes <- clinical %>% 
+  dplyr::filter(experimental_strategy == "Methylation", 
+                sample_type == "Tumor", 
+                composition == "Solid Tissue",
+                cohort != "TARGET",
+                dkfz_v12_methylation_subclass_score >= 0.8) %>% 
+  dplyr::mutate(molecular_subtype_methyl = 
+                  case_when(grepl("LGG_MYB", dkfz_v12_methylation_subclass) ~ "LGG, MYB/MYBL1",
+                            dkfz_v12_methylation_subclass == "SEGA" ~ "SEGA, To be classified",
+                            dkfz_v12_methylation_subclass == "A_IDH_LG" ~ "LGG, IDH",
+                            dkfz_v12_methylation_subclass == "PA_INF_FGFR" ~ "LGG, FGFR",
+                            dkfz_v12_methylation_subclass == "AG_MYB" ~ "LGG, MYB/MYBL1",
+                            dkfz_v12_methylation_subclass == "GG" ~ "GNG, MAPK")) %>% 
```

Comment:
Can you also add these:

Abbrevation | molecular_subtype_methyl
-- | --
A_IDH_LG | LGG, IDH
AG_MYB | LGG, MYB/MYBL1
DGONC | GNT
DIG_DIA | GNT
DLGNT_1 | GNT, MAPK
DLGNT_2 | GNT, MAPK
GG | GNG, MAPK
LGG_MYB_B | LGG, MYB/MYBL1
LGG_MYB_C | LGG, MYB/MYBL1
LGG_MYB_D | LGG, MYB/MYBL1
PA_INF_FGFR | LGG, FGFR
DNET | GNT
GNT_A | GNT
PA_CORT | GNT, MAPK
PA_INF | LGG, FGFR
PA_INF_FGFR | LGG, FGFR
PA_MID | LGG, MAPK
PLNTY | LGG, MAPK
RGNT | GNT, FGFR
PXA | PXA